RingPublishingGDPR/src/main/java/com/ringpublishing/gdpr/RingPublishingGDPRActivity.java

```diff
@@ -68,6 +69,16 @@ protected void onCreate(Bundle savedInstanceState)
         formView = RingPublishingGDPR.getInstance().getFormView(this.getApplicationContext());
 
         final LinearLayout layout = findViewById(R.id.content);
+        if(layout.getChildCount() > 0)
+        {
+            layout.removeView(formView);
+            layout.removeAllViews();
```

Comment:
a po co usuwasz osobno ten pojedynczy jak i tak potem wszystkie usuwasz?

Reply:
Jak się usuwa ten pojedyńczy, to też w nim usuwana jest referencja do parenta